include/morphio/section.h

```diff
@@ -42,46 +42,66 @@ class Section: public SectionBase<Section>
     /**
```

Comment:
can you also collapse the single line comments?

Reply:
Done